tests/yadism/cf/test_nc_kernels.py

```diff
@@ -54,10 +54,9 @@ def mkpv(nf, w):  # pv = parity violating
 def check(ps, w):
     assert len(w) == len(ps)
     for e, k in zip(ps, w):
-        assert pytest.approx(e) == k.partons
+        assert e == k.partons
```

Comment:
Are you sure you don't need `approx`? If you are comparing `float`s is always required, you should omit only for `int`s

Reply:
Most likely tests which were comparing floats are still skipped.
I wasn't sure how to restore also the `fonll_diff` ones.

Reply:
About FONLL diff I'd not worry that much, since #175 should remove it.